CHANGELOG.md

```diff
@@ -7,11 +7,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.1.1] - 2020-01-20
+
+### Added
+
+- Sass preprocessor integration.
```

Comment:
Section is fine but it should be under `Unreleased` header. We define tag when it comes the time to create one (release stage). It might be misleading that I added info about added changelog under `0.1.0` version but in fact I didn't pushed that tag. That was only example.